### PR TITLE
Support for environment variables for URL and TargetRepo

### PR DESCRIPTION
**Pull request by AlexeiVainshtein**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -35,5 +35,15 @@ func goCmd(c *cli.Context) error {
 		args = []string{""}
 	}
 
-	return gocmd.ExecuteGoCentral(strings.Join(args, " "))
+	// Check env first.
+	url := os.Getenv("GOC_GO_CENTER_URL")
```
Comment (eyalbe4) [inline main.go]: We may need add trailing slash to the URL if needed.
We actually have a util that does this. If this util is part of part of *jfrog-client-go*, we can probably use it here.

File: main.go [hunk 2]
```diff
@@ -35,5 +35,15 @@ func goCmd(c *cli.Context) error {
...
+	if url == "" {
+		url = "https://gocenter.jfrog.io/gocenter/"
+	}
+
+	targetRepo := os.Getenv("GOC_GO_CENTER_REPO")
+	if targetRepo == "" {
+		targetRepo = "gocenter-virtual"
+	}
+	return executers.Execute(strings.Join(args, " "), targetRepo, url)
```
Comment (eyalbe4) [inline main.go]: I think it makes more sense to sent the *url* argument before the *targetRepo* argument.
Actually, maybe *repo* name choice than *targetRepo* in this case.

**Comment by eyalbe4** [comment]
👍 @AlexeiVainshtein 
See my inline suggestions.

File: main.go (hunk 1 above)
Comment (AlexeiVainshtein) [reply main.go]: I'm using inside the GoCmd so GOC doesn't have the client as a dependency

File: main.go (hunk 2 above)
Comment (AlexeiVainshtein) [reply main.go]: Done

**Pull request merged**